schemas/glean/glean/glean.1.schema.json

```diff
@@ -3,6 +3,12 @@
   "$schema": "http://json-schema.org/draft-06/schema#",
   "additionalProperties": false,
   "description": "Schema for the ping content sent by Mozilla's glean telemetry SDK\n",
+  "mozPipelineMetadata": {
+    "bq_dataset_family": "",
+    "bq_metadata_format": "structured",
+    "bq_table": "",
```

Comment:
I can't remember why I set them in the first place; however I _think_ it's fine because it's an empty string and will still be overwritten. I double-check this though, it'd be nicer if we didn't have to specify them here.

Reply:
Ah yes, so:

this works because MSG for examples defines these defaults:

```
{'mozPipelineMetadata': {'bq_dataset_family': 'glean_core', 'bq_table': 'baseline_v1', 'bq_metadata_format': 'structured'}}
```
(exact values of course depending on the app/ping).

So when it gets the schema, with those empty strings, it still overwrites those keys with what it considers the defaults.

Now why did I set those to empty strings in the first place? Stay tuned.

Reply:
Ah, because https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/1db4caaf3717e9e9ed9e65d1cec51bd140073896/scripts/inject-metadata#L49:

Previously `mozPipelineData` was set to `false`, so `inject-metadata` did nothing.
Now that there is stuff in there, if I leave out `bq_dataset_family` and `bq_table` it will result in 

```
"bq_dataset_family": "glean",
"bq_metadata_format": "structured",
"bq_table": "glean_v1",
```

in there.
Why?
Because it generates some defaults here: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/1db4caaf3717e9e9ed9e65d1cec51bd140073896/scripts/inject-metadata#L62-L66

Merges those into the meta values we have defined in our `defaults.schema.json`, but only if they don't exist yet: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/1db4caaf3717e9e9ed9e65d1cec51bd140073896/scripts/inject-metadata#L67

And then injects them again: https://github.com/mozilla-services/mozilla-pipeline-schemas/blob/1db4caaf3717e9e9ed9e65d1cec51bd140073896/scripts/inject-metadata#L68-L69

---

Is this a problem? I don't know.
As seen we would still overwrite `bq_dataset_family` and `bq_table` later anyway, but I don't know if anything else depends on those values NOT being there.

Reply:
Ah so if you leave out `bq_dataset_family` and `bq_table`, they're going to be populated with
```
"bq_dataset_family": "glean",
"bq_metadata_format": "structured",
"bq_table": "glean_v1",
```
here, and therefore won't be overwritten in m-s-g?

Reply:
They will be overwritten in MSG actually; I tested this yesterday, see latest comment below.
Therefore I think we're fine not setting those here, downstream tools will see the correct values in the final schemas